mirage/config.js

```diff
@@ -39,10 +40,16 @@ export default function() {
     const { id } = request.params;
     const { me } = request.queryParams;
 
+    if (id) {
+      return schema.users.find(id); // users in the second case
+    }
+
     // check that this is a request for an auth'd user and there's a cookie
-    if (me && document.cookie.includes('token')) return schema.users.first();
+    if (me) {
```

Comment:
No more check for a cookie here? Why not? 
Should we also update the comment?

Reply:
@godfreyyeung Yep!

Reply:
@godfreyyeung because we're using http-only cookies, and because you can't simulate http-only cookies in mirage (because mirage is executed in the browser, which cannot touch http-only cookies), we just have to assume that the API handles the cookies correctly.